src/tls.rs

```diff
@@ -104,8 +104,8 @@ mod encryption {
                             }
                             #[cfg(feature = "rustls-tls-webpki-roots")]
                             {
-                                root_store.add_server_trust_anchors(
-                                    webpki_roots::TLS_SERVER_ROOTS.0.iter().map(|ta| {
+                                root_store.add_trust_anchors(
```

Comment:
The updated version requirements in Cargo.toml are needed because this function does not exist in older versions? If so, only the version of rustls has to be bumped or not? Why also the bump of tokio-rustls?

Reply:
Yes looks like `add_trust_anchors` was added in _rustls_ `0.21.6` and `add_server_trust_anchors` deprecated.

I don't know what changed in _tokio-rustls_ `0.24.1`. But it's probably harmless to up the min version anyway :shrug:

Reply:
If there's no reason then it forces users to update for no reason, and that might actually break something for them if for whatever reason they must use an older version. I wouldn't unnecessarily bump minimum versions.

Reply:
It looks like a refactor happened around commit `07e8da6e52f0eb76323f77d441bd1a06498b2f1c` [here](https://github.com/rustls/tokio-rustls/compare/main%40%7B07e8da6e52f0eb76323f77d441bd1a06498b2f1c%7D...main)'s the diff from `0.24.0` to `0.24.1` for tokio-rustls as I could find it.